index.js

```diff
@@ -26,6 +27,57 @@ export async function audience (pubSubMessage, context) {
   }
 }
 
+export async function acs (pubSubMessage, context) {
+  const s3 = new S3({
+    region: 'us-east-1', 
+    accessKeyId: process.env.S3_ACCESS_KEY_ID, 
+    secretAccessKey: process.env.S3_SECRET_ACCESS_KEY,  
```

Comment:
These ENV don't exist in `secrets` and aren't passed to the Google Cloud Function ([serverless/environment.js](https://github.com/CruGlobal/cru-udp-pipeline/blob/Tealium-To-ACS-Function/serverless/environment.js#L6-L15)). This includes `S3_BUCKET` ENV as well. I'll create a terraform PR that generates these values and adds them to `secrets`, but you'll need to make sure they get added to the environment.js file. Only keys in that file get passed the the Cloud Functions.